### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by anasanjuan**

### CODE REVIEW

File: staff/ana-sanjuan/app-template/home.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    </header>
+    <hr>
+
+    <section>
+        <article>
+            <header>
+                <img src="https://placekitten.com/200/200" alt="cute kitten">
+                <h2>Lorem title</h2>
+            </header>
+            <article>
```
Comment (diesttro) [inline staff/ana-sanjuan/app-template/home.html]: I think is confusing to put an article inside an article

Comment (anasanjuan) [reply staff/ana-sanjuan/app-template/home.html]: yes, I have been rethinking the code, and I think I need to use main and put sections inside instead of articles inside articles...  thanks for your comment!!

Comment (almuitur) [reply staff/ana-sanjuan/app-template/home.html]: Maybe article is duplicated?

Comment (anasanjuan) [reply staff/ana-sanjuan/app-template/home.html]: In the news page I do repeat the articles, but here I think is better to change to sections. 
Thanks for your comment Almudena! :)

File: staff/ana-sanjuan/app-template/about.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            <a href="home.html">Home</a> |
+            <a href="news.html">News</a> |
+            <a href="about.html">About</a> |
+            <a href="contact.html">Contact</a>|
+            <a href="register.html">Register</a> |
+            <a href="login.html">Login</a> |
+            <a href="profile.html">Profile</a> |
+        </nav>
+    </header>
+    <hr>
```
Comment (manuelbarzi) [inline staff/ana-sanjuan/app-template/about.html]: why HR here? if you just add because of presentational reasons, the remove it. presentation is not cared here in this challenge.

File: staff/ana-sanjuan/app-template/login.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <a href="news.html">News</a> |
+            <a href="about.html">About</a> |
+            <a href="contact.html">Contact</a>|
+            <a href="register.html">Register</a> |
+            <a href="login.html">Login</a> |
+            <a href="profile.html">Profile</a> |
+        </nav>
+    </header>
+    <hr>
+    <article>
```
Comment (manuelbarzi) [inline staff/ana-sanjuan/app-template/login.html]: why `article` wrapping a form here? it doesnt seem to have sense, please RTFM

File: staff/ana-sanjuan/app-template/profile.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Profile</title>
+</head>
+<body>
+    <header>
+        <h1>Register</h1>
```
Comment (manuelbarzi) [inline staff/ana-sanjuan/app-template/profile.html]: Register or Profile?

File: staff/ana-sanjuan/app-template/profile.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            <a href="about.html">About</a> |
+            <a href="contact.html">Contact</a>|
+            <a href="register.html">Register</a> |
+            <a href="login.html">Login</a> |
+            <a href="profile.html">Profile</a> |
+        </nav>
+    </header>
+    <hr>
+    <section>
+        <article>
```
Comment (manuelbarzi) [inline staff/ana-sanjuan/app-template/profile.html]: `article` here does not seem to have sense. RTFM

File: staff/ana-sanjuan/app-template/register.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+            <a href="news.html">News</a> |
+            <a href="about.html">About</a> |
+            <a href="contact.html">Contact</a>|
+            <a href="register.html">Register</a> |
+            <a href="login.html">Login</a> |
+            <a href="profile.html">Profile</a> |
+        </nav>
+    </header>
+    <hr>
+    <article>
```
Comment (manuelbarzi) [inline staff/ana-sanjuan/app-template/register.html]: `article`? no sense

File: staff/ana-sanjuan/skylab-precourse-projects/README.md [hunk 7]
```diff
@@ -11,16 +11,16 @@ I hope you enjoy them!
 ## Projects
```
Comment (manuelbarzi) [inline staff/ana-sanjuan/skylab-precourse-projects/README.md:12]: this file SHOULD NOT be included in this PR. you have mixed two branches into develop or you have been working both things in the same branch. please, take care and do not mix work again.

File: staff/ana-sanjuan/app-template/about.html (hunk 2 above)
Comment (anasanjuan) [reply staff/ana-sanjuan/app-template/about.html]: Ok!

File: staff/ana-sanjuan/app-template/login.html (hunk 3 above)
Comment (anasanjuan) [reply staff/ana-sanjuan/app-template/login.html]: I have deleted it here. thanks!

File: staff/ana-sanjuan/app-template/profile.html (hunk 4 above)
Comment (anasanjuan) [reply staff/ana-sanjuan/app-template/profile.html]: profile! thanks

File: staff/ana-sanjuan/app-template/profile.html (hunk 5 above)
Comment (anasanjuan) [reply staff/ana-sanjuan/app-template/profile.html]: I have read the FM, and  I now understand the difference between main, section and article! Thanks!

File: staff/ana-sanjuan/skylab-precourse-projects/README.md (hunk 7 above)
Comment (anasanjuan) [reply staff/ana-sanjuan/skylab-precourse-projects/README.md:12]: Yes, I merged two branches into develop, and I am afraid I did it intentionally. I thought it was the only way to update the projects if there was already an open pull request in develop... I see I was wrong, next time will ask before! thanks
